### PR TITLE
Remove references to `isNullObject` from Scripts docs

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

File: docs/docs-ref-autogen/excel/excelscript/excelscript.table.yml [hunk 1]
```diff
@@ -163,9 +163,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Table#getColumn:member(1)'
     package: ExcelScript!
     fullName: getColumn(key)
-    summary: >-
-      Gets a column object by name or ID. If the column doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a column object by name or ID. If the column doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.table.yml:166]:
```suggestion
    summary: 'Gets a column object by name or ID. If the column doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml [hunk 2]
```diff
@@ -1178,9 +1174,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Workbook#getSlicer:member(1)'
     package: ExcelScript!
     fullName: getSlicer(key)
-    summary: >-
-      Gets a slicer using its name or ID. If the slicer doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a slicer using its name or ID. If the slicer doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml:1177]:
```suggestion
    summary: 'Gets a slicer using its name or ID. If the slicer doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml [hunk 3]
```diff
@@ -1210,9 +1204,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Workbook#getSlicerStyle:member(1)'
     package: ExcelScript!
     fullName: getSlicerStyle(name)
-    summary: >-
-      Gets a `SlicerStyle` by name. If the slicer style doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a `SlicerStyle` by name. If the slicer style doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml:1207]:
```suggestion
    summary: 'Gets a `SlicerStyle` by name. If the slicer style doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml [hunk 4]
```diff
@@ -1242,9 +1234,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Workbook#getTable:member(1)'
     package: ExcelScript!
     fullName: getTable(key)
-    summary: >-
-      Gets a table by name or ID. If the table doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a table by name or ID. If the table doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml:1237]:
```suggestion
    summary: 'Gets a table by name or ID. If the table doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml [hunk 5]
```diff
@@ -1314,9 +1311,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Worksheet#getSlicer:member(1)'
     package: ExcelScript!
     fullName: getSlicer(key)
-    summary: >-
-      Gets a slicer using its name or ID. If the slicer doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a slicer using its name or ID. If the slicer doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml:1314]:
```suggestion
    summary: 'Gets a slicer using its name or ID. If the slicer doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml [hunk 6]
```diff
@@ -1392,9 +1387,7 @@ methods:
     uid: 'ExcelScript!ExcelScript.Worksheet#getTable:member(1)'
     package: ExcelScript!
     fullName: getTable(key)
-    summary: >-
-      Gets a table by name or ID. If the table doesn't exist, then this function will return an object with its
-      `isNullObject` property set to `true`<!-- -->.
+    summary: 'Gets a table by name or ID. If the table doesn''t exist, then this function returns `undefined`<!-- -->.'
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml:1390]:
```suggestion
    summary: 'Gets a table by name or ID. If the table doesn't exist, then this function returns `undefined`<!-- -->.'
```

File: docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml [hunk 7]
```diff
@@ -97,8 +97,7 @@ methods:
     fullName: getRange()
     summary: >-
       Returns the `Range` object that represents the range to which the AutoFilter applies. If there is no `Range`
-      object associated with the AutoFilter, then this method returns an object with its `isNullObject` property set to
-      `true`<!-- -->.
+      object associated with the AutoFilter, then this method returns `undefined`<!-- -->.
```
Comment (alison-mk) [inline docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml:100]: This description uses "method", but all the others use "function". I recall you changing function to method at one point (in a different PR related to isNullObject). What are your thoughts on that variation here?

**Review by alison-mk — CHANGES_REQUESTED** [review]
One question and some typo fixes.

File: docs/docs-ref-autogen/excel/excelscript/excelscript.table.yml (hunk 1 above)
Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel/excelscript/excelscript.table.yml:166]: These are escape characters generated by the tooling for one-line description support. They don't render in the resulting markdown: https://review.docs.microsoft.com/en-us/javascript/api/office-scripts/excelscript/excelscript.table?branch=AlexJ-Intellisense11-11&view=office-scripts#getColumn_key_

File: docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml (hunk 7 above)
Comment (AlexJerabek) [reply docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml:100]: It's an unintentional difference, but one that's in numerous other places in the docs. This Considering the amount of upstream effort for this one change, I'd rather fix the method/function distinction in the base metadata.

**Review by alison-mk — APPROVED** [review]

**Pull request merged**